src/Compilers/CSharp/Portable/CSharpParseOptions.cs

```diff
@@ -67,7 +67,7 @@ public override IEnumerable<string> PreprocessorSymbolNames
             this.SpecifiedLanguageVersion = languageVersion;
             this.LanguageVersion = languageVersion.MapSpecifiedToEffectiveVersion();
             this.PreprocessorSymbols = preprocessorSymbols.ToImmutableArrayOrEmpty();
-            _features = features?.ToImmutableDictionary() ?? ImmutableDictionary<string, string>.Empty;
+            _features = features?.ToImmutableDictionary(StringComparer.OrdinalIgnoreCase) ?? ImmutableDictionary<string, string>.Empty;
```

Comment:
@AlekseyTs That may certainly be the case, but so far we haven't been able to find the cause despite good communication from the affected developer. Since it's important to get him unblocked, it leads me to question the downsides of what we know to be a potential workaround. :smile: